src/coreComponents/python/modules/geosx_mesh_tools_package/geosx_mesh_tools/abaqus_converter.py

```diff
@@ -0,0 +1,124 @@
+
+import meshio
+from meshio._mesh import CellBlock
+import numpy as np
+import argparse
+import logging
+
+
+def convert_abaqus_to_gmsh(input_mesh, output_mesh, verbose=False):
+    """
+    @brief Convert an abaqus mesh to gmsh 2 format, preserving nodeset information
+    @param input_mesh path of the input abaqus file
+    @param output_mesh path of the output gmsh file
+    @param verbosity flag to display optional logging information
+    """
+    # Open the logger
+    logging.basicConfig(level=logging.WARNING)
+    logger = logging.getLogger(__name__)
+    if verbose:
+      logger.setLevel(logging.INFO)
+
+    # Load the mesh
+    logger.info('Reading abaqus mesh...')
+    mesh = meshio.Mesh.read(input_mesh, "abaqus")
+
+    # Convert the element regions to tags
+    logger.info('Converting region tags...')
+    region_list = list(mesh.cell_sets.keys())
+    n_regions = len(region_list)
+    cell_ids = np.zeros(len(mesh.cells[0][1]), dtype=int) - 1
```

Comment:
Probably add a comment that indicates that this `-1` value is a wrong value must disappear?